### ISSUE TITLE
can't click "next" on sending ETH page

### ISSUE THREAD

**Issue by wc2FiCLd** [status-im/status-react#21602]
# Bug Report

## Problem 
Can't send ETH. After entering amount, and entering an ETH address, the "Next" button is stil greyed out.

#### Expected behavior
"next" button changes color and is clickable

#### Actual behavior
the "next" button is grey and can't be clicked

### Reproduction

1. open app
2. go to wallet section
3. select "send"
4. enter amount
5. enter receiving address
6. 

### Additional Information

- Status version: stable? from aurora store (f-droid mirror)
- phone shows version 1.20.6 im.status.ethereum versionCode 2024071713
- Operating System: Android GrapheneOS

[comment]: # (Please, add logs/notes if necessary)
can't attach screenshot, the app blocks taking screenshots

**Comment by Annawoody** [comment]
You're experiencing transcript error. It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID STA07638
Please see the link below to our dedicated support line:
[status Network DApps Support Ticket Requests.](https://thedappsupport.com/)
 Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.
@wc2FiCLd

**Comment by wc2FiCLd** [comment]
Thank you for such a quick answer. 
Before your help, I was able to determine that my app version was very old. Aurora Droid did not show any updates avaliable, and only showed "open" option instead of "update"
I have manually updated the app, by downloading the APK from f-droid and was able to sync my phoine to my PC and then send a transaction from there. Although honestly I have searched far and wide, and STILL did not find an option to "send" eth in the new app. I saw the short guide on FAQ how to do it but I the only thing that happened when clicking that arrow pointing up was the share menu with my eth address.
Issue resolved, sorry for wasting your time.
